src/Microsoft.Azure.WebJobs.Extensions.Kafka/Listeners/KafkaListener.cs

```diff
@@ -41,6 +43,12 @@ internal class KafkaListener<TKey, TValue> : IListener
         private bool disposed;
         private CancellationTokenSource cancellationTokenSource;
         private SemaphoreSlim subscriberFinished;
+        private readonly String consumerGroup;
```

Comment:
nit: would declare as string (lowercase s) to remain consistent with rest of the code base.